src/main/java/org/gaul/s3proxy/S3ProxyHandler.java

```diff
@@ -425,6 +425,9 @@ private void doHandle(HttpServletRequest request,
             if (headerName.startsWith("x-amz-meta-")) {
                 continue;
             }
+            if (headerName.startsWith("x-amz-content-")) {
+                continue;
+            }
```

Comment:
No. For Azure backend, this caused an error `Unknown header x-amz-content-sha256 with URI`. Sorry I should not have mixed this here. Let me know if I should create a different PR for this or another way to fix it.